great_expectations/data_context/types/base.py

```diff
@@ -700,6 +700,40 @@ def validate_schema(self, data, **kwargs):
             raise ge_exceptions.InvalidConfigError(
                 f"""Your current configuration uses one or more keys in a data connector that are required only by an
 SQL type of the data connector (your data connector is "{data['class_name']}").  Please update your configuration to
+continue.
+                """
+            )
+        if (
+            "data_asset_name_prefix" in data
+            or "data_asset_name_suffix" in data
+            or "excluded_tables" in data
+            or "included_tables" in data
+        ) and not (
+            data["class_name"]
+            in [
+                "InferredAssetSqlDataConnector",
+                "ConfiguredAssetSqlDataConnector",
+                "InferredAssetGlueCatalogDataConnector",
+                "ConfiguredAssetGlueCatalogDataConnector",
+            ]
+        ):
+            raise ge_exceptions.InvalidConfigError(
+                f"""Your current configuration uses one or more keys in a data connector that are required only by an
+SQL/GlueCatalog type of the data connector (your data connector is "{data['class_name']}").  Please update your configuration to
+continue.
+                """
+            )
+
+        if ("catalog_id" in data or "glue_introspection_directives" in data) and not (
```

Comment:
@lccasagrande The inclusion of `catalog_id` and `glue_introspection_directives` looks appropriate, but it also seems like there may be some merge issues, because the validation of other fields ought to have been there before.

Reply:
Done

Reply:
❤️ @lccasagrande Looks like you added another useful check for all SQL `DataConnector` implementations!  If that is correct, big thank you!!!